odoo/tools/image.py

```diff
@@ -49,7 +49,7 @@
 
 class ImageProcess():
 
-    def __init__(self, base64_source, verify_resolution=True):
+    def __init__(self, base64_source, grayscale=False, verify_resolution=True):
```

Comment:
grayscale should be an operation that you call in the chain, not a param of the constructor